src/apps/activity_assignments/api.py

```diff
@@ -41,6 +66,51 @@ async def assignments_create(
     )
 
 
+async def unassignments_create(
+    applet_id: uuid.UUID,
+    user: User = Depends(get_current_user),
+    schema: ActivitiesAssignmentsCreate = Body(...),
+    session=Depends(get_session),
```

Comment:
Please use another class to unassign, using a `ActivityAssignmentsCreate` to "delete" is very strange

Reply:
ActivityAssignmentsDelete class created